cypress/e2e/class_management/showGroupsAndClassesInTable.feature

```diff
@@ -1,27 +1,34 @@
+#@release
 @unstable_test
 Feature: Group - To show groups and classes in one table with respective functionality
 
   As a teacher I want to see all groups and classes belonging to my school.
 
-
-  Scenario: As a pre-condition teacher adds a class to school
+  Scenario: As a teacher i can add a class to school
     Given I am logged in as a 'teacher1_nbc' at 'nbc'
```

Comment:
we need maybe also tests for other instances.. just to make sure that the feature flag is working